autogpt/commands/web_selenium.py

```diff
@@ -64,6 +64,12 @@ def scrape_text_with_selenium(url: str) -> Tuple[WebDriver, str]:
     options.add_argument(
         "user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.5615.49 Safari/537.36"
     )
+    options.add_argument(
+        '--no-sandbox'
+    )
+    options.add_argument(
+        '--headless'
+    )
```

Comment:
Just enabling headless unconditionally is not a good idea. Same for the sandbox.

Reply:
I didn't know that. Maybe if I add environment variables (as actually was suggested on discord in a discussion earlier) for both, that would work, right?